README.md

```diff
@@ -1,4 +1,126 @@
-# sacd
+# Service Access Contract Definition (SACD)
+
+## SACD JSON Format
+
+This document provides a basic overview of the SACD JSON format, which is used to define agreements for exchanging services and assets within the DIMO ecosystem. SACDs ensure that involved parties have a mutually agreed-upon understanding of the exchange and its scope, including any compensation. These agreements are designed to be human-readable and can represent various types of agreements, including permissions and payments.
+
+The [`sacd.template.json`](data/sacd.template.json) provides a generic structure for SACD JSON files. Example SACD files for payments ([`sacd.payment.example.json`](data/sacd.payment.example.json)) and permissions ([`sacd.permission.example.json`](sacd.permission.example.json)) demonstrate how this template is used for specific use cases.
+
+### Overview
+
+The SACD JSON follows a consistent structure, as seen in the template:
+
+```json
```

Comment:
lets have this match what we have in token-exchange PR https://github.com/DIMO-Network/token-exchange-api/pull/29/files#diff-1a26f007d96312c258da05d0a6d4ef05fe0027ed9bbeff513b5faef48f8f8f04R65
so no additionalInfo{} since we do not need it and it's not hard to add later.